examples/tests/invalid/map_entries.test.w

```diff
@@ -0,0 +1,2 @@
+let map = { 1 => "foo", 2 => "bar" };
```

Comment:
This will result in a lot more errors than just `^ Expected type to be "str", but got "num" instead` since maps expect keys to be strings. 

If thats not intentional then lets change it to
```suggestion
let map = { "1" => "foo", "2" => "bar" };
```